src/Compilers/VisualBasic/Portable/VisualBasicParseOptions.vb

```diff
@@ -29,7 +29,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' <param name="kind">The kind of source code.<see cref="SourceCodeKind"/></param>
         ''' <param name="preprocessorSymbols">An enumerable sequence of KeyValuePair representing preprocessor symbols.</param>
         Public Sub New(
-            Optional languageVersion As LanguageVersion = LanguageVersion.VisualBasic14,
+            Optional languageVersion As LanguageVersion = LanguageVersion.VBnext,
```

Comment:
@AnthonyDGreen Was planning to revert this back to `VisualBasic14` once I got the unit tests to work.